ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientFactory.java

```diff
@@ -171,13 +229,44 @@ public static ManagedStreamingIngestClient createManagedStreamingIngestClientFro
     public static ManagedStreamingIngestClient createManagedStreamingIngestClientFromDmCsb(ConnectionStringBuilder dmConnectionStringBuilder,
             @Nullable HttpClientProperties properties)
             throws URISyntaxException {
-        return ManagedStreamingIngestClient.fromDmConnectionString(dmConnectionStringBuilder, properties);
+        return ManagedStreamingIngestClient.fromDmConnectionString(dmConnectionStringBuilder, properties, true);
+    }
+
+    /**
```

Comment:
Never make new code that is deprecated.
The whole point of deprecated is not supported.
Don't add any variants to deprecated code.